### ISSUE TITLE
Super cool module!

### ISSUE THREAD

**Issue by michaellwest** [doodleM/SitecoreReactIntegration#1]
I am curious how this is related to the module created by the esteemed community member @guitarrich and this [repo](https://github.com/GuitarRich/sitecore.react). Is it possible that this repository should be a fork of his and some credit given to the creator?

**Comment by jammykam** [comment]
![sitecore react package](https://user-images.githubusercontent.com/1770296/30763521-984a3bcc-9fb4-11e7-8d58-e6e72a5ea0f9.png)

Seems to be a clone, the module package still has Richard's name in there.

**Comment by jst-cyr** [comment]
Agreed. A fork would be a better way to do this. If this is planned as a larger module that uses Rich's module as a dependency, it might be better to pull it in as a package that you are referencing so that if the original changes the user can get the latest from the source and then apply your extensions on top of it.

(Side note, probably better to use the public Sitecore NuGet feeds to reference the Sitecore dlls to make it easier to switch versions.)
